src/backend/drm/surface/atomic.rs

```diff
@@ -90,18 +90,9 @@ impl State {
         // If they don't match, `commit_pending` will return true and they will be changed on the next `commit`.
         let mut current_connectors = HashSet::new();
         // make sure the mapping is up to date
-        map_props(fd, res_handles.connectors(), &mut prop_mapping.0)?;
+        map_props(fd, res_handles.connectors(), &mut prop_mapping.connectors)?;
```

Comment:
If we want be consistent here, we should probably clear `prop_mapping.connectors` first to not keep old entries around of properties missing for a new connector, no?